components/switch/switch-mixin.js

```diff
@@ -4,8 +4,9 @@ import { css, html } from 'lit-element/lit-element.js';
 import { getUniqueId } from '../../helpers/uniqueId.js';
 import { ifDefined } from 'lit-html/directives/if-defined.js';
 import { RtlMixin } from '../../mixins/rtl-mixin.js';
+import { FocusVisiblePolyfillMixin } from '../../mixins/focus-visible-polyfill-mixin.js';
 
-export const SwitchMixin = superclass => class extends RtlMixin(superclass) {
+export const SwitchMixin = superclass => class extends RtlMixin(FocusVisiblePolyfillMixin(superclass)) {
```

Comment:
I'd recommend the `eslint` VS Code (or other IDE) plugin as it'll catch these as you go!

Reply:
I also ended up getting the `stylelint` extension, very helpful thanks